src/util.py

```diff
@@ -40,6 +40,12 @@ def __init__(self) -> None:
                 )
             )
 
+    @staticmethod
+    def get_languages() -> list[str] | None:
+        if len(get_settings().LANGUAGES) == 0:
```

Comment:
Empty string is falsy:
```suggestion
        if not get_settings().LANGUAGES:
```